third-party/libtm/libtm/include/TrackingCommon.h

```diff
@@ -136,7 +136,7 @@ namespace perc {
         SIXDOF_MODE_FAST_PLAYBACK = 0x0001,
         SIXDOF_MODE_ENABLE_MAPPING = 0x0002,
         SIXDOF_MODE_ENABLE_RELOCALIZATION = 0x0004, 
-        SIXDOF_MODE_MAX
+        SIXDOF_MODE_MAX = ((SIXDOF_MODE_FAST_PLAYBACK | SIXDOF_MODE_ENABLE_MAPPING | SIXDOF_MODE_ENABLE_RELOCALIZATION) + 1)
```

Comment:
Why not `SIXDOF_MODE_MAX = 0x0008`, or  
`SIXDOF_MODE_MAX  = SIXDOF_MODE_ENABLE_RELOCALIZATION<<1`  ?